Android build job is failing

The [android build job is failing](https://app.circleci.com/pipelines/github/pytorch/vision/9483/workflows/94e80aa1-0cb5-49e8-9ed6-ae3be328016f/jobs/699884) with:

```
2021-07-16T08:46:31.505+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-07-16T08:46:31.509+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Could not determine the dependencies of task ':test_app:lintVitalFrcnnMnetv3BaseLocalRelease'.
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Could not resolve all artifacts for configuration ':test_app:frcnnMnetv3CameraLocalDebugCompileClasspath'.
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not find org.pytorch:pytorch_android:1.9.0-SNAPSHOT.
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]      Required by:
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :test_app
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not find org.pytorch:pytorch_android_torchvision:1.9.0-SNAPSHOT.
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]      Required by:
2021-07-16T08:46:31.510+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]          project :test_app
```

Perhaps we should update this line?

https://github.com/pytorch/vision/blob/d993ce568576306168d24ee597fefa3d8ba40998/android/build.gradle#L18:L18

Or perhaps something's wrong with the upstream pytorch mobile builds?

@IvanKobzarev since you authored https://github.com/pytorch/vision/pull/3679, would you mind taking a look? Thanks!!

cc @seemethere

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -15,7 +15,7 @@ allprojects {
             androidSupportAppCompatV7Version = "28.0.0"
             fbjniJavaOnlyVersion = "0.0.3"
             soLoaderNativeLoaderVersion = "0.8.0"
-            pytorchAndroidVersion = "1.9.0-SNAPSHOT"
+            pytorchAndroidVersion = "1.10.0-SNAPSHOT"
         }
 
         repositories {

